### PR TITLE
Remove supported flags from feature gate description.

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

**Describe what this PR does**
With the latest k8s, we dont need to enable some of the mentioned feature gates by command line.

**Is there anything that requires special attention?**
Do you have any questions? Did you do something clever?

**Related issues:**
Mention any github issues relevant to this PR

### CODE REVIEW

**Comment by humblec** [comment]
@Madhu-1 PTAL.

File: README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ clusterrolebinding.rbac.authorization.k8s.io/glusterfs-csi-role created
 Below listed feature gates need to be enabled in kubernetes v1.12.1
 
 ```
---feature-gates=CSIPersistentVolume=true,MountPropagation=true,VolumeSnapshotDataSource=true,KubeletPluginsWatcher=true,CSINodeInfo=true,CSIDriverRegistry=true
+--feature-gates=VolumeSnapshotDataSource=true,"CSINodeInfo=true","CSIDriverRegistry=true"
```
Comment (Madhu-1) [inline README.md]: do we require double quotes? if yes we need to add it to `VolumeSnapshotDataSource=true` also

Comment (humblec) [reply README.md]: yeah,  let me add.

Comment (humblec) [reply README.md]: @Madhu-1 Done.

Comment (Madhu-1) [reply README.md]: @humblec, I don't think we need to add double quotes around feature gates
example from the kubernetes doc
`--feature-gates=BlockVolume=true,CSIBlockVolume=true`

Comment (humblec) [reply README.md]: I think the effect is same, however I removed the quotes for parity between the docs. @Madhu-1 ptal.

**Comment by humblec** [comment]
@JohnStrunk PTAL ..

**Pull request merged**